sdk/cognitivelanguage/Azure.AI.Language.Conversations/api/Azure.AI.Language.Conversations.netstandard2.0.cs

```diff
@@ -1,258 +1,19 @@
-    }
-    public partial class BaseExtraInformation
-    {
-        internal BaseExtraInformation() { }
-    }
-    public partial class BasePrediction
-    {
-        internal BasePrediction() { }
-        public Azure.AI.Language.Conversations.ProjectKind ProjectKind { get { throw null; } set { } }
-        public string TopIntent { get { throw null; } }
-    }
-    public partial class BaseResolution
-    {
-        internal BaseResolution() { }
-    }
-    public partial class BooleanResolution : Azure.AI.Language.Conversations.BaseResolution
-    {
-        internal BooleanResolution() { }
-        public bool Value { get { throw null; } }
-    }
     public partial class ConversationAnalysisClient
     {
         protected ConversationAnalysisClient() { }
         public ConversationAnalysisClient(System.Uri endpoint, Azure.AzureKeyCredential credential) { }
         public ConversationAnalysisClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.AI.Language.Conversations.ConversationAnalysisClientOptions options) { }
+        public virtual System.Uri Endpoint { get { throw null; } }
         public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual Azure.Response AnalyzeConversation(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
-        public virtual Azure.Response<Azure.AI.Language.Conversations.AnalyzeConversationTaskResult> AnalyzeConversation(string utterance, Azure.AI.Language.Conversations.ConversationsProject project, Azure.AI.Language.Conversations.AnalyzeConversationOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> AnalyzeConversationAsync(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.Language.Conversations.AnalyzeConversationTaskResult>> AnalyzeConversationAsync(string utterance, Azure.AI.Language.Conversations.ConversationsProject project, Azure.AI.Language.Conversations.AnalyzeConversationOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public static string GetAuthorizationHeader() { throw null; }
+        public virtual Azure.Operation<System.BinaryData> CancelJob(Azure.WaitUntil waitUntil, System.Guid jobId, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> CancelJobAsync(Azure.WaitUntil waitUntil, System.Guid jobId, Azure.RequestContext context = null) { throw null; }
         public virtual Azure.Response JobStatus(System.Guid jobId, bool? showStats = default(bool?), Azure.RequestContext context = null) { throw null; }
```

Comment:
Did architects approve the name of this method?  It is not a verb.

Reply:
They haven't reviewed these APIs yet. How is CancelJob i.e. "cancel job" not a verb? Pairs nicely with SubmitJob i.e. "submit job".

Reply:
I was referring to "JobStatus" with this comment.